<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationForm</class>
 <widget class="QMainWindow" name="RegistrationForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1226</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1226</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1226</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Bank system</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(239, 239, 239);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="Name">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Telephone">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>250</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Email">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>340</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Surname">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="PasportSer">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>70</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Cancle">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>520</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
background-color: rgba(255, 119, 121, 255);
</string>
    </property>
    <property name="text">
     <string>Отмѣнить</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PasportNum">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>160</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ConfirmPassword">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>430</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="Password">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>340</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Background">
    <property name="geometry">
     <rect>
      <x>152</x>
      <y>29</y>
      <width>921</width>
      <height>571</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Name_lable">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Имя</string>
    </property>
   </widget>
   <widget class="QLabel" name="Surname_lable">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Фамилія </string>
    </property>
   </widget>
   <widget class="QLabel" name="Telephone_lable">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>220</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Телефонъ</string>
    </property>
   </widget>
   <widget class="QLabel" name="Email_lable">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>310</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Почтовый ящикъ</string>
    </property>
   </widget>
   <widget class="QLabel" name="PassportNum_lable">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>130</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Номеръ паспорта </string>
    </property>
   </widget>
   <widget class="QLabel" name="PassportSer_lable">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>40</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Серія паспорта</string>
    </property>
   </widget>
   <widget class="QLabel" name="Password_lable">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>310</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Шифръ </string>
    </property>
   </widget>
   <widget class="QLabel" name="ConfirmPassword_lable">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>400</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Подтвердите шифръ </string>
    </property>
   </widget>
   <widget class="QLabel" name="Background_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>-110</y>
      <width>811</width>
      <height>851</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../Images.qrc">:/LoginImage/resource/images/Coat_of_arms_of_Russia_Empire_without_shield.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Submit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>520</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
background-color: rgb(170, 255, 164);
</string>
    </property>
    <property name="text">
     <string>Записать</string>
    </property>
   </widget>
   <widget class="QLabel" name="Company_lable">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>400</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Предпріятіе</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Company">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>430</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="international">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>210</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Иностранный подданный</string>
    </property>
   </widget>
   <zorder>Background_2</zorder>
   <zorder>Background</zorder>
   <zorder>Telephone</zorder>
   <zorder>Email</zorder>
   <zorder>Surname</zorder>
   <zorder>PasportSer</zorder>
   <zorder>Cancle</zorder>
   <zorder>PasportNum</zorder>
   <zorder>ConfirmPassword</zorder>
   <zorder>Password</zorder>
   <zorder>Name</zorder>
   <zorder>Name_lable</zorder>
   <zorder>Surname_lable</zorder>
   <zorder>Telephone_lable</zorder>
   <zorder>Email_lable</zorder>
   <zorder>PassportNum_lable</zorder>
   <zorder>PassportSer_lable</zorder>
   <zorder>Password_lable</zorder>
   <zorder>ConfirmPassword_lable</zorder>
   <zorder>Submit</zorder>
   <zorder>Company_lable</zorder>
   <zorder>Company</zorder>
   <zorder>international</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1226</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../Images.qrc"/>
 </resources>
 <connections/>
</ui>
